packages/gen-typescript-declarations/src/cli.ts

```diff
@@ -53,6 +54,12 @@ const argDefs = [
         'writing new typings, excluding node_modules/, bower_components/, ' +
         'or any file added using the <addReferences> config option.',
   },
+  {
+    name: 'verify',
+    type: Boolean,
+    description: 'Compile the generated types with TypeScript and fail if ' +
```

Comment:
Lets add the version in the description to be super clear and prevent confusion later.

Reply:
Done.